### PR TITLE
Feature patient feedback

### PR DESCRIPTION
**Pull request by MirkoVojinovicRA182**

### CODE REVIEW

**Review by MitarBrankovic — APPROVED** [review]

File: manager-application/src/app/feedbacks/all-feedback-view/all-feedback-view.service.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AllFeedbackViewService {
+
+  private _url = "http://localhost:16934/api/"
```
Comment (tara-pogancev) [inline manager-application/src/app/feedbacks/all-feedback-view/all-feedback-view.service.ts]: Hardcoded URL

File: patient-application/src/app/feedback.service.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import { Observable } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+import { Content } from '@angular/compiler/src/render3/r3_ast';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class RandomNumberGeneratorService {
+
+  private _url = "http://localhost:16934/api/"
```
Comment (tara-pogancev) [inline patient-application/src/app/feedback.service.ts]: Hardcoded URL, might be good to define them all in a separate file for easier later fixes

File: HospitalInformationSystem/HospitalAPI/Controllers/FeedbackController.cs [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+namespace HospitalAPI.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class FeedbackController : ControllerBase
+    {
+        private static readonly string[] Summaries = new[]
+        {
+            "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
```
Comment (tara-pogancev) [inline HospitalInformationSystem/HospitalAPI/Controllers/FeedbackController.cs]: Leftover code?

**Review by tara-pogancev — CHANGES_REQUESTED** [review]
package-lock.json; add to Angular gitignore

File: HospitalInformationSystem/HospitalAPI/Controllers/FeedbackController.cs (hunk 3 above)
Comment (MitarBrankovic) [reply HospitalInformationSystem/HospitalAPI/Controllers/FeedbackController.cs]: Done.

File: patient-application/src/app/feedback.service.ts (hunk 2 above)
Comment (tara-pogancev) [reply patient-application/src/app/feedback.service.ts]: Refactored

File: manager-application/src/app/feedbacks/all-feedback-view/all-feedback-view.service.ts (hunk 1 above)
Comment (tara-pogancev) [reply manager-application/src/app/feedbacks/all-feedback-view/all-feedback-view.service.ts]: Refactored

**Review by tara-pogancev — APPROVED** [review]
Looks good! 🚀

**Review by vladimirr9 — APPROVED** [review]

**Pull request merged**